installing.adoc

```diff
@@ -96,59 +103,70 @@ Found zimbra-store (local)
 Found zimbra-apache (local)
 Found zimbra-spell (local)
 Found zimbra-convertd (local)
-Found zimbra-imapd (local)
 Found zimbra-memcached (repo)
 Found zimbra-proxy (local)
 Found zimbra-archiving (local)
 Found zimbra-chat (repo)
 Found zimbra-drive (repo)
+Found zimbra-imapd (local)
+Found zimbra-network-modules-ng (local)
 
 
 Use Zimbra's package repository [Y] y
 
 Use internal development repo [N] n
 Configuring package repository
 
-Select the packages to install
+Install zimbra-ldap [Y] y
 
-Install zimbra-ldap [Y] Y
+Install zimbra-logger [Y] y
 
-Install zimbra-logger [Y] N
+Install zimbra-mta [Y] y
 
-Install zimbra-mta [Y] N
+Install zimbra-dnscache [Y] n
 
-Install zimbra-dnscache [Y] N
+Install zimbra-snmp [Y] y
 
-Install zimbra-snmp [Y] N
+Install zimbra-store [Y] y
 
-Install zimbra-store [Y] N
+Install zimbra-apache [Y] y
 
-Install zimbra-apache [Y] N
+Install zimbra-spell [Y] y
 
-Install zimbra-spell [Y] N
+Install zimbra-convertd [Y] y
 
-Install zimbra-convertd [Y] N
+Install zimbra-memcached [Y] y
 
-Install zimbra-memcached [Y] N
+Install zimbra-proxy [Y] y
 
-Install zimbra-proxy [Y] N
+Install zimbra-archiving [N] y
 
-Install zimbra-archiving [N] N
+Install zimbra-chat [Y] y
 
-Install zimbra-chat [Y] N
+Install zimbra-drive [Y] y
 
-Install zimbra-drive [Y] N
+Install zimbra-imapd [Y] n            
 
-Install zimbra-imapd [Y] Y
+Install zimbra-network-modules-ng [Y] y
 
-.
-.
+###WARNING###
+
+Network Modules NG needs to bind on TCP ports 8735 and 8736 in order
+to operate, for inter-instance communication.
+Please verify no other service listens on these ports and that 
```

Comment:
Please verify that no other service...

Reply:
This is a comment on the own product installation, I will not edit it on the documentation.